src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dayRent = 40;
+  let totalCost = dayRent * days;
+
+  if (days >= 3 && days < 7) {
```

Comment:
2. [checklist.md] Don't use literals in expressions (HARDCODE). Instead use constants explaining this values